src/Twig/Extension/SettingsExtension.php

```diff
@@ -25,6 +25,7 @@ public function getFunctions(): array
             new TwigFunction('kbin_captcha_enabled', [SettingsExtensionRuntime::class, 'kbinCaptchaEnabled']),
             new TwigFunction('kbin_mercure_enabled', [SettingsExtensionRuntime::class, 'kbinMercureEnabled']),
             new TwigFunction('kbin_federation_page_enabled', [SettingsExtensionRuntime::class, 'kbinFederationPageEnabled']),
+            new TwigFunction('kbin_sidebar_sections_local_only', [SettingsExtensionRuntime::class, 'kbinSidebarSectionsLocalOnly']),
```

Comment:
very minor, but it doesn't look like this gets used in any twig template (this comment applies to [this function as well](kbinSidebarSectionsLocalOnly)). Now that I look, I don't think `kbinFederatedSearchOnlyLoggedIn()` gets called anywhere either, but that's not something related to your PR. I imagine the overhead on an unused twig function isn't high, so this isn't a big issue if true, but just wanted to mention it. These extensions expose the function to be usable in twig templates but I don't think we have a use case for it as the setting seems contain inside the backend at the moment